src/pages/SubmissionDetails.js

```diff
@@ -136,27 +189,19 @@ export const SubmissionDetails = ({ submissionId }) => {
                 }
               },
               {
-                size: { basis: 155, grow: 0 },
-                field: 'workflow_params',
-                headerRenderer: () => 'Data',
+                size: { basis: 550, grow: 0 },
+                field: 'run_id',
+                headerRenderer: () => 'Run ID',
                 cellRenderer: ({ rowIndex }) => {
                   return div({ style: { width: '100%', textAlign: 'left' } }, [
-                    h(Link, { style: { display: 'inline-block', marginBottom: '1rem', textDecoration: 'underline' }, onClick: () => setViewInputsId(rowIndex) }, ['View inputs']),
-                    h(Link, { style: { display: 'inline-block', marginTop: '1rem', textDecoration: 'underline' }, onClick: () => setViewOutputsId(rowIndex) }, ['View outputs'])
-                  ])
-                }
-              },
-              {
-                size: { basis: 250, grow: 0 },
-                field: 'logs',
-                headerRenderer: () => 'Logs',
-                cellRenderer: ({ rowIndex }) => {
-                  return div({ style: { width: '100%', textAlign: 'center' } }, [
-                    h(Link, { disabled: true, onClick: () => setViewInputsId(rowIndex) }, ['View workflow log file'])
+                    h(Link, { style: { display: 'inline-block', textDecoration: 'underline' }, onClick: () => setViewInputsId(rowIndex) }, [paginatedPreviousRuns[rowIndex].run_id])
```

Comment:
On click could copy the deleted action above, ie `{ onClick: () => { Nav.goToPath('workflow-dashboard', { workflowId: paginatedPreviousRuns[rowIndex].engine_id }) }, style: { fontWeight: 'bold' } },`